git-link.el

```diff
@@ -917,24 +924,27 @@ With a prefix argument prompt for the remote's name.
 Defaults to \"origin\"."
 
   (interactive (list (git-link--select-remote)))
-  (let* (commit handler remote-info (remote-url (git-link--remote-url remote)))
+  (let* ((remote-url (git-link--remote-url remote))
+         commit handler remote-info git-host web-host)
```

Comment:
Any reason for reversing these?

Reply:
If I remember correctly, when I added `git-host` and `web-host` the line became very long and looked cleaner when split. Then it seemed more idiomatic to place the variables without an initial value at the end, but I don't think it was necessary in any way :thinking: